lib/users.rb

```diff
@@ -0,0 +1,17 @@
+class Users
+  def self.create(username:, password:)
+    connection = Database.connect
+    result = connection.exec("SELECT username FROM users WHERE username = '#{username}'")
+    if result.ntuples != 0
+      return false
+    end
+    connection.exec("INSERT INTO users (username, password) VALUES('#{username}','#{password}')")
```

Comment:
For securing the app to avoid SQL injection via input field, use exec_params.

connection.exec_params(
  "INSERT INTO users (username, password) VALUES($1, $2) RETURNING id, username, password;", [sername, password]
)